### PR TITLE
CONTRIBUTING.rst: Add different versions of docs

### PR DESCRIPTION
**Pull request by mr-karan**
Fixes https://github.com/hylang/hy/issues/1033

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -11,6 +11,13 @@ Pull requests are great! We love them; here is a quick guide:
   contribute but don't know how to begin, the `good-first-bug`_ label 
   of the `issue tracker`_ is the place to go. 
   (If you're new to Git: `Start Here`_)
+  
+- Before contributing make sure you checkout the docs. There are two versions of docs available
```
Comment (tuturto) [inline CONTRIBUTING.rst]: Just a small change. I would write "make sure you check the docs.".

Comment (kirbyfan64) [inline CONTRIBUTING.rst]: And don't forget the colon at the end of the line!

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -11,6 +11,13 @@ Pull requests are great! We love them; here is a quick guide:
   contribute but don't know how to begin, the `good-first-bug`_ label 
   of the `issue tracker`_ is the place to go. 
   (If you're new to Git: `Start Here`_)
+  
+- Before contributing make sure you checkout the docs. There are two versions of docs available 
+
+  + `latest`_, for use with bleeding-edge github version.
```
Comment (kirbyfan64) [inline CONTRIBUTING.rst]: with **the** bleeding-edge

(Also, do people normally capitalize GitHub?)

Comment (Kodiologist) [inline CONTRIBUTING.rst]: Ryan, while I'm thankful that you're looking out for typos, I think that when it comes to minor changes like writing issues, it's better to provide a patch than to write comments. (You can also push directly to the author's branch if the author has enabled that, which is the default, I think.)

**Comment by mr-karan** [comment]
Sorry, I messed up this branch in my local git and wasn't able to rebase properly. i have opened a new PR and made the changes request. https://github.com/hylang/hy/pull/1164 

Apologies again !

**Comment by tuturto** [comment]
No problem at all.

**Pull request closed without merging**